sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -14,7 +14,7 @@
     requires transitive org.reactivestreams;
 
     requires transitive org.slf4j;
-
+    requires transitive java.logging;
```

Comment:
Why do we require either of these to be transitive? Do we have any public APIs which require an SLF4J of Java logging object?

Reply:
@JonathanGiles

Reply:
I commented earlier that this should no longer be necessary given it isn't being used any longer.

Reply:
Yes. Removed.